src/database/models/Payments.ts

```diff
@@ -0,0 +1,30 @@
+import { Schema, model } from "mongoose";
+
+
+
+enum Status {
```

Comment:
we can create a separate `enums` folder and create a file for the `Status` enum, that way we can reduce cluttering in the files

Reply:
when you do that, you also probably want to create functions that can convert from enum to string (and vice versa) so we can easily store on the DB

Reply:
yup toString() works, and i created a seperate enum folder as well